.github/workflows/go.yml

```
@@ -0,0 +1,67 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  gosec:
+    name: Gosec scan
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Run Gosec Security Scanner
+        run: | # https://github.com/securego/gosec/issues/469
+          export PATH=$PATH:$(go env GOPATH)/bin
+          go install github.com/securego/gosec/v2/cmd/gosec@v2.12.0
+          gosec -exclude=G307
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.18
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

[non-blocking][question] does `lint` intentionally use a previous version of Go than `build`? If not, proposal to update to Go 1.19 here?

```suggestion
      - name: Set up Go 1.19
        uses: actions/setup-go@v3
        with:
          go-version: 1.19
```

Motivation: this repo 1.19 in `go.mod` so when maintainers bump to subsequent versions of Go, a <kbd>CMD</kbd>+<kbd>F</kbd> finds all occurences of the currently configured Go version.

Ref: https://github.com/ralexstokes/relay-monitor/blob/main/go.mod#L3